tests/contrib/pylons/test_pylons.py

```diff
@@ -65,3 +70,39 @@ def test_pylons():
     assert s.duration <= end - start
     eq_(s.error, 0)
     eq_(s.meta.get(http.STATUS_CODE), '200')
+
+def test_pylons_exceptions():
+    writer = DummyWriter()
+    tracer = Tracer()
+    tracer.writer = writer
+    app = FakeWSGIApp()
+    traced = PylonsTraceMiddleware(app, tracer, service="p")
+
+    # successful request
+    assert not writer.pop()
+    app.code = '200 OK'
+    app.body = ['woo']
+    app.environ = {
+        'REQUEST_METHOD':'GET',
+        'pylons.routes_dict' : {
+            'controller' : 'foo',
+            'action' : 'bar',
+        }
+    }
+
+    try:
+        out = traced(app.environ,  app.start_response_exception)
+    except Exception as e:
+        pass
+
+    assert not tracer.current_span(), tracer.current_span().pprint()
+    spans = writer.pop()
+    assert spans, spans
+    eq_(len(spans), 1)
+    s = spans[0]
+
+    eq_(s.error, 1)
+    eq_(s.get_tag("error.msg"), "Some exception")
+    sc = int(s.get_tag("http.status_code"))
+    assert sc >= 100 and sc < 600
```

Comment:
you should check the exact status code you're expecting and not the range, otherwise we can't get if the right status code has been updated or not